test/exercism/comment_test.rb

```diff
@@ -26,7 +26,7 @@ def test_mentions
       ["Mention @alice and @bob and also @charlie", %w(alice bob), "ignore unknown users"],
     ].each do |body, mentions, _desc|
       usernames = User.where(id: Comment.new(body: body).mention_ids).map(&:username)
-      assert_equal usernames, mentions
+      assert_equal usernames.sort, mentions.sort
```

Comment:
In general, I think this is fine. Another way to test this is just to make sure both arrays include the same values regardless of order (probably could be done a few ways but one way is to just minus the arrays and check that there's nothing left).